src/Workspaces/VisualBasic/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.vb

```diff
@@ -369,12 +370,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                                 Return True
                             End If
 
-                            ' If both the expression and it's parent are binary expressions and their kinds
-                            ' are the same, check to see if they are commutative (e.g. + or *).
-                            If parentBinaryExpression.IsKind(SyntaxKind.AddExpression, SyntaxKind.MultiplyExpression) AndAlso
+                            ' If both the expression And its parent are binary expressions And their kinds
```

Comment:
Typos: `And`, `Is`, `New`, `Not` and `hte`

Reply:
Done. this is what happens when you paste a C# comment into VB :)  #resolved